index.html

```diff
@@ -117,37 +117,30 @@ <h2>Network requests</h2>
 
       <p>
       A <dfn data-lt="network requests">network request</dfn> occurs when the
-      user agent must use the network to service a single <dfn data-cite="RFC7230#section-2.1">request</dfn>.
+      user agent must use the network to service a single [=request=].
       </p>
 
       <p>
-      If the user agent can service a <a>request</a> out of a local
-        cache, that <a>request</a> MUST NOT result in a <a>network
-        request</a>.
+      If the user agent can service a [=request=] out of a local cache, that
+      [=request=] MUST NOT result in a <a>network request</a>.
       </p>
 
       <p>
-      If the user agent follows [=redirect status|redirects=] as part of a [=navigate|navigation=],
-      there MUST be separate <a>network requests</a> for each <a>request</a> in
-      the redirect chain.
+      If the user agent follows [=redirect status|redirects=] as part of a

```

Comment:
Same.

Reply:
I think what you're suggesting is to delete these two paragraphs, because (HTTP-network) fetch solves that for us. Correct?